lua/crates/completion/common.lua

```diff
@@ -138,10 +204,33 @@ local function complete()
     local line, col = util.cursor_pos()
     local crates = util.get_line_crates(buf, Span.new(line, line + 1))
     local _, crate = next(crates)
+
+    if state.cfg.completion.crates.enabled then
+        local working_crates = state.buf_cache[buf].working_crates
+        for _,wcrate in ipairs(working_crates) do
+            if wcrate and wcrate.span:moved(0, 1):contains(col) then
```

Comment:
- This is missing a check if we're in the right line.
- `span` should probably be named `col`.
- the `wcrate` nil check is redundant